### DISCUSSION TITLE
Tabs.List There is a problem on the small screen.

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2256] [Q&A]
**Tabs.List**
There is a problem on the small screen.
How to solve the problem

<img width="553" alt="Screen Shot 2565-08-28 at 06 53 43" src="https://user-images.githubusercontent.com/12431734/187051864-96e4b019-83a4-4055-8418-0613e44e2a83.png">

**Comment by ryuhangyeong** [comment]
## Suggest

It seems to be helpful in solving the problem if you provide a minimum testable example (ex. code pen)

**Reply by goodlythink** [reply]
https://codesandbox.io/s/dreamy-dijkstra-vb3tyn?file=/src/App.tsx

It can be solved by using ScrollArea in Card
but i want it Tab.List responsive start new line

**Reply by rtivital** [reply]
https://codesandbox.io/s/clever-mirzakhani-bxgwy3?file=/src/App.tsx

**Comment by zhulien-ivanov** [comment]
I guess this can be closed as https://github.com/mantinedev/mantine/commit/4e6dd62505a47757fd568e2475bd1a49ec54a4c4 is also merged in master.
